.github/ISSUE_TEMPLATE/release-a-new-version.md

```diff
@@ -1,30 +1,32 @@
-title: ''
+about: Publishing a new release
+title: 'Releas a New Version'
 labels: ''
 assignees: ''
 
 ---
 <!-- Please Follow this checklist before making your release. Thanks! -->
 
-### Release Checklist
-
+## Publish a New Release
+### Release Preparation
 - [ ] Decide on a version number based on the current version number and the common rules defined in [Semantic Versioning](https://semver.org). E.g. `2.0.x`.
 - [ ] Update this ticket name to reflect the new version number, following the pattern "Release `2.0.x`".
 - [ ] Update `version` string (Eg: `2.0.x`) attribute in the `build.gradle.kts`
-- [ ] Cut a branch for the new release out of the `develop` branch, following the Gitflow naming pattern `release/2.0.x`.
-- [ ] Update the code to use this version number.
+- [ ] Code freeze and cut a branch for the new release out of the `develop` branch, following the Gitflow naming pattern `release/2.0.x`.
 - [ ] Update the badges versions in [docs/00 - Stellar Anchor Platform.md].
-- [ ] Make all changes necessary to make sure the release is ready to be published. If new issues are found during the manual tests, create new tickets aiming at improving the automated tests so these issues can be automatically detected next time.
+- [ ] In general, only bug fixes and security patches will be be applied to the `release/2.0.x` branch.
+### Release Publication
 - [ ] DO NOT RELEASE before holidays or weekends! Mondays and Tuesdays are preferred.
-- [ ] Create a new release on GitHub with the name `2.0.x` and tag: `2.0.x` (without the `release-` prefix).
-  - [ ] Check the docker image of the release automatically published to [Docker Hub](https://hub.docker.com/r/stellar/anchor-platform).
-  - [ ] Update the badges versions in [docs/00 - Stellar Anchor Platform.md].
-- [ ] Create two pull requests:
-  - [ ] `release/2.0.x -> main`: this should require two approvals.
-  - [ ] `release/2.0.x -> develop`: ideally, this should be merged after the `main` branch is merged.
-  - [ ] (Optional) You'll need to manually publish a new version of the SDK to [Maven Central](https://search.maven.org/search?q=g:org.stellar.anchor-sdk).
-  - [ ] (Optional) You'll need to manually upload the jar file from [Maven Central](https://search.maven.org/search?q=g:org.stellar.anchor-sdk) to the GH release.
+- [ ] Create a new release draft on GitHub with the name `2.0.x` and tag: `2.0.x` (without the `release-` prefix).
+- [ ] Write the proper release notes.
+- [ ] After reviewing the release draft, publish!!!
+### Post Release Publication
+- [ ] Check the docker image of the release automatically published to [Docker Hub](https://hub.docker.com/r/stellar/anchor-platform).
+- [ ] If necessary, update the badges versions in [docs/00 - Stellar Anchor Platform.md].
+- [ ] Create the pull request `release/2.0.x -> main`: this should require two approvals.
+- [ ] Create another pull request `main -> develop`: AFTER the release branch is merged with the `main` branch.
```

Comment:
Can you also add that this merges must be regular, not squash merges?

Reply:
Added